_scripts/capslock-out.txt

```diff
@@ -0,0 +1,40 @@
+Capslock is an experimental tool for static analysis of Go packages.
```

Comment:
I think this file should be deleted now, if I'm not missing anything only the capslock_platform_output.txt files are used.

Reply:
Done.